### ISSUE TITLE
ND not refreshing properly

### ISSUE THREAD

**Issue by ErikNL80** [flybywiresim/a32nx#6920]
### Aircraft Version

Stable

### Build info

```json
{
  "built": "2022-01-08T01:22:50+00:00",
  "ref": "v0.7.4",
  "sha": "5adc0dc1ee3bd9cf9386c312b798a5f8ca7b391b",
  "actor": "Benjozork",
  "event_name": "manual"
}
```

### Describe the bug

Since a few days I experience the ND not refreshing properly anymore. The compass and ground speed are not refreshed, TCAS and waypoints seems to be moving properly. Also the flightplan line is not drawn, only the waypoints are visible. During the flight the flightplan line returns and the ND seems to be behaving.

### Expected behavior

I would expect all parts of the ND to refresh properly...

### Steps to reproduce

1. Start a flight
2. Observe the ND on the ground
3. Look for gorund speed, compass and flightplan

### References (optional)

I have some screenshots from a flight from Bristol to Innsbruck. It shows what is happening...

This is after refreshing the ND by switching it off... No flightplan is shown.
![EGGD](https://user-images.githubusercontent.com/71258178/158330950-1e382702-6eed-472b-847e-616db9247083.png)

This is when standing still and after a 90 degrees turn...
![EGGD2](https://user-images.githubusercontent.com/71258178/158330956-6e10e5dd-8145-4580-bd31-6b64f0f9f2a6.png)

This is before take-off at runway 09 (see the compass still showing 27 on the ND)...
![EGGD3](https://user-images.githubusercontent.com/71258178/158330961-96d659ac-ae27-45cc-b53d-ea06e4579f39.png)

This is after passing 10.000ft (normal again):
![EGGD4](https://user-images.githubusercontent.com/71258178/158331528-a9353e8f-c223-44f6-bbfc-0d3328e42fb2.png)

Refresh rate of the gauges in the sim is set to 'Medium'.

### Additional info (optional)

Could it be related to Sim Update 8?

**Comment by tracernz** [comment]
This is the old Asobo ND which has been entirely replaced, so unlikely we will work on it.

**Comment by ErikNL80** [comment]
According to the installer, compatibility is guaranteed with each majopr patch of MSFS. Will there be any new version soon then?

**Comment by 2hwk** [comment]
Should be fixed as of 0.8.0
